vignettes/Variation.Rmd

```diff
@@ -1,66 +1,90 @@
+### Plotting functions
+First, we will create a few plotting functions to visualise outputs. 
+```{r}
+cols <- c("#E69F00", "#56B4E9", "#009E73", 
+          "#F0E442", "#0072B2", "#D55E00", "#CC79A7")
+          
+plot_n_detect <- function(output, ylab=TRUE){
+  if (ylab==TRUE) {
+    ylab = "Cases detected in children aged 2-10 years"
+    } else {ylab=''}
+  plot(x = output$timestep, y = output$n_detect_730_3650, 
+       type = 'l', col = cols[3],
+       xlab = 'Time (days)', ylab = ylab,
+       xaxs = "i", yaxs = "i", bty = "l")
+  grid(lty = 2, col = 'grey80', lwd = 1)
+}
+```
 
+## Parameterisation
+First, we will use the `get_parameters()` function to generate a list of parameters, accepting the default values, for two different population sizes. Then we will update the parameters to match equilibrium parameters and to achieve the initial EIR using `set_equilibrium()`
 ```{r}
-# A small population
-p <- get_parameters(list(
-  human_population = 1000,
+# A small population 
+simparams <- get_parameters(list(
+  human_population = 100,
   individual_mosquitoes = FALSE
 ))
-p <- set_equilibrium(p, 2)
-small_o <- run_simulation(365, p)
-p <- get_parameters(list(
+
+simparams <- set_equilibrium(simparams, init_EIR = 20)
+
+# A larger population 
+simparams <- get_parameters(list(
   human_population = 10000,
   individual_mosquitoes = FALSE
 ))
-p <- set_equilibrium(p, 2)
-big_o <- run_simulation(365, p)
-
-plot_grid(
-  ggplot(small_o) + geom_line(aes(timestep, n_detect_730_3650)) + ggtitle('n_detect at n = 1,000'),
-  ggplot(small_o) + geom_line(aes(timestep, p_detect_730_3650)) + ggtitle('p_detect at n = 1,000'),
-  ggplot(big_o) + geom_line(aes(timestep, n_detect_730_3650)) + ggtitle('n_detect at n = 10,000'),
-  ggplot(big_o) + geom_line(aes(timestep, p_detect_730_3650)) + ggtitle('p_detect at n = 10,000')
-)
+
+simparams <- set_equilibrium(simparams, init_EIR = 20)
 ```
 
-The n_detect output shows the result of sampling individuals who would be detected by microscopy.
 
-While the p_detect output shows the sum of probabilities of detection in the population.
+## Simulations
+
+The `n_detect_730_3650` output below shows the total number of individuals in the age group rendered (here, 730-3650 timesteps or 2-10 years) who have microscopy-detectable malaria. Notice that the output is smoother at a higher population.
 
-Notice that the p_detect output is slightly smoother. That's because it forgoes the sampling step. At low population sizes, p_detect will be smoother than the n_detect counterpart.
+```{r, fig.align='center',fig.height=4, fig.width=10}
+# A small population
+output_small_pop <- run_simulation(365, simparams)
+
+# A larger population
+output_big_pop <- run_simulation(365, simparams)
+
+# Plotting 
+par(mfrow = c(1,2))
+plot_n_detect(output_small_pop); title('n_detect at n = 100')
+plot_n_detect(output_big_pop, ylab = FALSE); title('n_detect at n = 10,000')
```

Comment:
Here it is a little hard to see the difference between the two plots immediately - I think changing to a prevalence output will help, and you could also force the y axes to cover the same range between both plots so it is more obvious and we are comparing same to same